projects/ad77681evb/zed/Makefile

```diff
@@ -1,5 +1,5 @@
 ####################################################################################
-## Copyright 2020(c) Analog Devices, Inc.
+## Copyright 2018(c) Analog Devices, Inc.
```

Comment:
why 2018?

Reply:
Our Makefile generator script has a static license header. Feel free to suggest a solution, if you think this is a problem. My point of view is "I don't know, I'm not a lawyer". :)

Reply:
fair enough 🍺